src/HomePage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Container, ListGroup, Form } from 'react-bootstrap';
+import { fetchCoins } from './redux/coins/coins';
+import { fetchGloblalData } from './redux/global/global';
+import Coin from './Coin';
+import GlobalCryptoData from './GlobalData';
+
+const HomePage = () => {
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that each coin list on the Homepage is behaving as a clickable object when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.

Reply:
Thank's  @rloterh  for all the feedback! I have done every change! I hope it will be approved nowl😄